### DISCUSSION TITLE
How I can save data?

### DISCUSSION THREAD

**Discussion by KamilM1205** [dodorare/crossbow#82] [Q&A] [answered]
I'm using macroquad + crossbow and I want make saves in game. How I can do it?

**Comment by enfipy** [comment, marked as answer]
Hey @KamilM1205!

We have plans on adding this feature through some official [Apple](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/GameKit_Guide/SavedGames/SavedGames.html) and [Google](https://developers.google.com/games/services/common/concepts/savedgames) approach. And of course, you always can just save it locally with `android-ndk` and `AAssetManager`.
